### PR TITLE
Improve Swedish translation

### PR DESCRIPTION
**Pull request by enoch85**
cc @small1

### CODE REVIEW

**Comment by enoch85** [comment]
ICLA is already signed.

File: lang/main-sv.json [hunk 1]
```diff
@@ -1211,11 +1276,13 @@
...
         "toggleCamera": "Byta kamera",
+        "unmute": "Slå på ljud",
         "videoSettings": "Video inställningar",
-        "videomute": "Aktivera / avaktivera kameran"
+        "videomute": "Aktivera / avaktivera kameran",
+        "videounmute": "Stoppa kameran"
     },
     "transcribing": {
-        "ccButtonTooltip": "Starta / Avsluta undertexter",
+        "ccButtonTooltip": "Atkivera / avaktivera undertexter",
```
Comment (Ayko1595) [inline lang/main-sv.json:1285]: Hello, I just stumbled on this PR and noticed that you have a typo here 😊

Comment (enoch85) [reply lang/main-sv.json:1285]: Well spotted! Thanks!

**Comment by enoch85** [comment]
cc @damencho Please have a look. Thanks!

**Comment by saghul** [comment]
The CI is still failing. You can check the failing action output for clues on what's wrong, or do "npm run lint:ci" to run it locally.

**Comment by enoch85** [comment]
@saghul Ran local tests now and they passed. Please approve CI

**Comment by enoch85** [comment]
Tested again with the current code and everything checks out:

```
danh@9320-RL:~/jitsi-meet/lang$ npm run lint:ci

> jitsi-meet@0.0.0 lint:ci
> eslint --ext .js,.ts,.tsx --max-warnings 0 .

danh@9320-RL:~/jitsi-meet/lang$ 

```

Is there anything else I can do to get this merged?

**Comment by damencho** [comment]
Jenkins, test this please.

**Pull request merged**